indico/modules/events/registration/templates/management/regform_modify.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'layout/base.html' %}
+
+{% block title %}
```

Comment:
Modification shouldn't be the title, but rather Registration, which comes with `_regform_base.html`.